docs/examples/postponed_annotations_works.py

```diff
@@ -1,10 +1,10 @@
 from __future__ import annotations
-from typing import List  # <-- List is defined in the module's global scope
+from typing import Any, List
```

Comment:
this comment is import for context, please revert.

Also better to use `int` to keep the meaning of the example as clear as possible.

Reply:
agreed, but waiting for response in https://github.com/pydantic/pydantic/pull/4339#discussion_r941665874

Reply:
Replaced `List` and `Any` with `HttpUrl` and added a comment